pyproject.toml

```diff
@@ -140,6 +140,7 @@ jquery = { version = "==3.7.0", include = ["jquery(.min)?(.js|.map)?$"] }
 font-awesome = { version = "==5.15.4", include = ["css/all(.min)?.css", "webfonts/*"] }
 bootstrap = { version = "==4.6.2", include = ["js/bootstrap.bundle(.min)?.js(.map)?", "css/bootstrap(.min)?.css(.map)?", "scss/*"] }
 bootswatch = { version = "==5.3.0", provider = "unpkg", include = ["dist/flatly/.*.scss"] }
+floating-scroll = { version = "==3.2.0", provider = "unpkg", include = ["dist/jquery.floatingscroll.min.js", "dist/jquery.floatingscroll.css"] }
 "@github/markdown-toolbar-element" = { version = "==1.2.0", provider = "unpkg", include = ["dist/index.umd.js$"] }
 # Use the builder at https://datatables.net/download/
 "datatables.net" = { version = "==1.11.3", provider = "unpkg", include = ["js/jquery.dataTables(.min)?.js"]}
```

Comment:
Do we still need `datatables.net-responsive` and `datatables.net-responsive-bs4`?

Reply:
I had not realized it was actually an extension instead of a build-in. Removed!